packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerScenario.ts

```diff
@@ -177,31 +222,49 @@ export function runUnitTestScenario(
 		const joiners: TestEditManager[] = [];
 		/**
 		 * Local helper to update all the state that is dependent on the sequencing of new edits.
+		 * Note that all commits are part of the same bunch and have the same session ID, sequence number
+		 * and reference sequence number.
 		 */
-		const recordSequencedEdit = (commit: TestCommit): void => {
-			trunk.push(commit.seqNumber);
+		const recordSequencedEdits = (commits: TestCommit[]): void => {
+			commits.forEach((commit) => {
+				trunk.push({ intention: commit.intention, seq: commit.seqNumber });
```

Comment:
It would be good to add some asserts to enforce that `commit.sessionId`, `commit.seqNumber`, and `commit.refNumber` are all the same.